packages/smart/src/hooks.ts

```diff
@@ -121,6 +121,13 @@ export const useSmart = <T extends Smart>(
   },
   options?: UseSmartOptions
 ): T => {
+  if (!window.sessionStorage.getItem("isKernelInitialized")) {
+    setTimeout(() => {
```

Comment:
I remember we discussed about this change, jsut to check via container. Not sure where we pushed it?